docs/conf.py

```diff
@@ -44,8 +61,9 @@
     'python': ('https://docs.python.org/3', None),
     'numpy': ('https://docs.scipy.org/doc/numpy', None),
     'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
-    'pandas': ('http://pandas.pydata.org/pandas-docs/stable/', None),
-    'astropy': ('http://docs.astropy.org/en/stable/', None)}
+    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
+    'astropy': ('http://docs.astropy.org/en/stable/', None),
+    'numba': ('https://numba.pydata.org/numba-doc/dev/', None)}
```

Comment:
Oh!  This is where these things are defined!

Reply:
Yup :) I'm sorry that this pr does all these little changes in irrelevant places - the joy of refactoring caught me.

Reply:
No worries!  Don't tell anyone that I'm doing that a bit in #728 too!  :scream_cat:  I don't mind us adding on little fixes here and there, since otherwise these little fixes might not get done.  And this time it had the side benefit that I got to learn a little more about the code!